src/EnergyPlus/SingleDuct.cc

```diff
@@ -4829,6 +4863,27 @@ namespace SingleDuct {
 			}
 			TestCompSet( cCurrentModuleObject, SysATMixer( ATMixerNum ).Name, cAlphaArgs( 5 ), cAlphaArgs( 4 ), "Air Nodes" );
 
+			if ( SysATMixer( ATMixerNum ).OARequirementsPtr == 0 ) {
+				if ( ZoneSizingInput.allocated( ) ) {
+					for ( int SizingInputNum = 1; SizingInputNum <= NumZoneSizingInput; ++SizingInputNum ) {
+						if ( ZoneSizingInput( SizingInputNum ).ZoneNum == SysATMixer( ATMixerNum ).ZoneNum ) {
+							SysATMixer( ATMixerNum ).SizingInputNum = SizingInputNum;
```

Comment:
This works, but you don't really need to store SysATMixer.SizingInputNum.  Just set SysATMixer.OARequirementsPtr = ZoneSizingInput( SizingInputNum ).ZoneDesignSpecOAIndex.